subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -98,40 +118,47 @@ static int get_published_payload(struct mqtt_client *client, u8_t *write_buf,
  * @param[in] fota_status next_state  The next state the device's fota operation
  *				      is going to enter. Reported to the status
  *				      details of the Job Execution.
- * @param[in] version_number  The expected version number of the Job Execution.
  * @param[in] client_token  Client identifier which will be repeated in the
  *			    respone of the update.
  *
  * @return 0 If successful otherwise a negative error code is returned.
  */
-#define AWS_FOTA_STATUS_DETAILS_TEMPLATE "{\"nextState\":\"%s\"}"
+#define NUM_OF_QUOTATION_MARKS 10
+#define AWS_FOTA_STATUS_DETAILS_TEMPLATE "{\"nextState\":\"%s\", \"offset\":%d}"
 #define STATUS_DETAILS_MAX_LEN  (sizeof("{\"nextState\":\"\"}") \
-				+ (sizeof("download_firmware") + 2))
+				+ (sizeof("download_firmware") \
+				+ (sizeof("offset") + NUM_OF_QUOTATION_MARKS)))
 
 static int update_job_execution(struct mqtt_client *const client,
 				const u8_t *job_id,
 				enum execution_status state,
 				enum fota_status next_state,
-				int version_number,
-				const char *client_token)
+				int offset,
+				const char *client_token
+				)
 {
+	wait_for_update_accepted();
```

Comment:
Add a comment explaining that this is waiting for the previous update (previous call to this function).

Reply:
Added.